### ISSUE TITLE
Pre-Defined Kits using assets instead of models

### ISSUE THREAD

**Issue by sg25dev** [grokability/snipe-it#10495]
### Snipe-IT Version

5.1.8

### Operating System

Ubuntu

### Web Server

Apache

### PHP Version

7.4

### Is your feature request related to a problem? Please describe. A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]

We have a requirement to group assets together as part of kits.  We planned on using the pre-defined kits option to do this, however it's not exactly what we need.  The pre-defined kits allow you to create a kit based on **asset models** that exist within the system.  We would like to have the ability to create kits based on **assets** that are in the system.

A pre-defined kit then becomes a set of specific **assets** rather than a set of **asset models**.

### Describe the solution you'd like A clear and concise description of what you want to happen.

The ability to create a pre-defined kit using assets as opposed to asset models.  If this does not work, the creation of a new element (asset group?) that would allow us to group specific assets together and to be able to checkin / checkout that group of assets together using one checkin / checkout.  Ideally, like assets, visibility to these kits could be limited by company.  IE if company X has created asset group ABC, company Y would not see asset group ABC when system configuration "Full Multiple Companies Support" is enabled.

### Describe alternatives you've considered A clear and concise description of any alternative solutions or features you've considered.

We have considered adding a custom field called "Kit Name" and adding a kit name here and adding the same kit name to the other assets that are part of this kit.  The checkin / checkout wouldn't be as easy then as it is with pre-defined kits since each item would need to be checked out individually.

**Comment by cswinton147** [comment]
We do this by creating a dummy asset as the "kit" and checkout all the items in the kit to this asset.
Then we can just checkout/checking the kit to a user without having to checkin/checkout each item

**Comment by sg25dev** [comment]
This is a very good suggestion.  Thank you for your input.  I will test this out and see if it meets our needs.  Thank you very much for sharing.

**Comment by spencer-ze** [comment]
I support this feature. It would streamline the checkin/checkout process of hundreds of assets we have packed in physical traveling kits.

**Comment by echristley** [comment]
We are using Snipe-IT for QA resource management.  We  run tests through Jenkins, and use the API to checkout devices.  Many of the tests only need one device, and Snipe-IT works admirably for that.  But some devices are hardwired together and need to communicate.  Checking them out to a dummy device is problematic, because there is no API to query what devices a device has checked out.  So, I'm not able to retrieve the device details of all of the devices.

**Comment by qvjester** [comment]
Yes really could do with assets in kits rather than asset models.

The dummy work around isn't really a good solution as it causes issues elsewhere.

**Comment by inventiontech** [comment]
would also love to see this feature available
